grails-app/assets/javascripts/streama/services/api-service.js

```diff
@@ -98,7 +98,7 @@ angular.module('streama').factory('apiService', function ($http, $rootScope, con
 				return $http.delete('file/removeFileFromDisk.json', {params: {id: id, path: path}});
 			},
       removeMultipleFilesFromDisk: function(bulk) {
-        return $http.delete('file/removeMultipleFilesFromDisk.json', {params: {id: bulk}})
+        return $http.delete('file/removeMultipleFilesFromDisk.json', {params: {id: bulk}});
```

Comment:
haha I think I very often added these in out of habit and removed them again before committing because I realized that they shouldn't be in the same commit, but here I forgot apparently :D